src/hooks/use-archive-site.ts

```diff
@@ -97,7 +94,7 @@ export function useArchiveSite() {
 					return;
 				}
 
-				const file = new File( [ archiveContent ], 'loca-env-site-1.zip', {
+				const file = new File( [ fs.readFileSync( archivePath ) ], 'loca-env-site-1.zip', {
```

Comment:
I think we cannot use `fs` from a React hook or component. We'll need to always run `fs.readFileSync` from a NodeJS file.

Reply:
Yep, I realized the blunder when unit tests failed. It's a mystery how it even seemed to work when it was tested 🤣 This is why I removed the review request and put the PR back in draft mode. I just force pushed commits, so should be sorted now.